scripts/utils.py

```diff
@@ -2,56 +2,55 @@
 from ape_safe import ApeSafe
 from gnosis.safe.safe_tx import SafeTx
 from brownie import Contract
-
-def fetch_current_nonce(safe: ApeSafe):
-    # small abi of safe, only to be able to check nonce
-    nonce_abi = [{
-        "stateMutability": "view",
-        "type": "function",
-        "name": "nonce",
-        "inputs": [],
-        "outputs": [
-                {
-                    "name": "",
-                    "type": "uint256"
-                }
-        ]
-    }]
-    return Contract.from_abi("GnosisSafeProxy", safe.address, nonce_abi).nonce()
+import urllib.request
+import json
 
 
 def confirm_posting_transaction(safe: ApeSafe, safe_tx: SafeTx):
     safe_nonce = safe_tx.safe_nonce
 
-    current_nonce = safe_tx.current_nonce
+    # fetch list of txs from gnosis api
+    url = "https://safe-transaction.mainnet.gnosis.io/api/v1/safes/0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE/multisig-transactions/"
```

Comment:
We need to generate the url based on the network we are on.
Base url and gnosis safe address will be different on each network.